docs/hosting-a-server-with-northstar/dedicated-server/best-practices.md

```diff
@@ -17,14 +16,12 @@ The current minimum requirements are as follows:
 
 - 5GB free disk space (+3 GB if you're on windows due to origin and deps)
 - 3+ cores (2 might work, though)
-- 6GB+ total memory (RAM or swap will do) 
-- Windows 8.1 (and built-in WARP [use `-softwared3d11`] or a graphics card) + or wine 6.0.0+ (and dxvk+lavapipe+x11, wined3d+llvmpipe+x11, or a working graphics setup with dxvk or llvmpipe) (see https://git.io/JyrOR for wine details)
-
+- 6GB+ total memory (RAM or swap will do)
+- Windows 8.1 (and built-in WARP [use `-softwared3d11`] or a graphics card) + or wine 6.0.0+ (and dxvk+lavapipe+x11, wined3d+llvmpipe+x11, or a working graphics setup with dxvk or llvmpipe) (see [this](https://git.io/JyrOR) for more)
```

Comment:
We should probably just unshorten the link by replacing `https://git.io/JyrOR` with `https://github.com/pg9182/northstar-dedicated`.